### PR TITLE
Driver test unit, big pull. Look through carefully! Was a bit messy.

### PR DESCRIPTION
**Pull request by rsbrost**
Needs thorough examination.

### CODE REVIEW

**Review by plesiopterys — COMMENTED** [review]
Reviewed basically all files except "drivers_test_unit.py" 

Andy should still validate other changes (e.g. to instrument_driver and setup.py requirements)

File: pyscan/drivers/stanford830.py
Comment (i-am-mounce) [inline pyscan/drivers/stanford830.py:1]: sensitivity should be voltage_sensitivity and current_sensitivity if splitting, but then they both depend on the indexed value on instrument. Not clear how to do this at the moment.

File: setup.py
Comment (i-am-mounce) [inline setup.py:1]: This does not cause an issue with running jupyter notebooks? In a new python instance, is this sufficient to run a jupyter notebook.

File: test/drivers/test_instrument_driver.py
Comment (i-am-mounce) [inline test/drivers/test_instrument_driver.py:1]: I don't think a values property should have tuples or lists in them.  The assumption for values property is that the value is simple and can be formatted as a string such as '{}'.format(value).  I don't think this works straightforwardly for tuples and lists.

Comment (i-am-mounce) [reply test/drivers/test_instrument_driver.py:1]: Also ranges property should not be inluded.  This is a niche chase that I don't want to include right now.

**Comment by rsbrost** [comment]
Note, a most recent commit includes changes to the keithley from the drivers branch so that we will be ready to test it completely.

**Comment by rsbrost** [comment]
Currently passing on keithly and stanford via 1111 pyscan machine.

**Review by i-am-mounce — APPROVED** [review]

**Pull request merged**